setup.py

```diff
@@ -33,13 +45,7 @@
     namespace_packages=['collective'],
     include_package_data=True,
     zip_safe=False,
-    install_requires=[
-        'setuptools',
-        'simplejson',
-        'collective.transmogrifier',
```

Comment:
Uhm... I thougth it was a good idea because `collective.transmogrifier` is anyway pulled in by `plone.app.transmogrifier`...
But now that I think twice I have some doubts about it... explicit is better than implicit!
@mauritsvanrees, do you suggest to have an explicit dependence on `collective.transmogrifier`?

In this case I will modify this PR without any problem :)